debian.enterprise/postinst

```diff
@@ -23,9 +23,9 @@ fi
 sed -i /etc/arangodb3/arangod.conf -e "s;storage-engine = auto;storage-engine = $STORAGE_ENGINE;"
 
 if [ -d /run/systemd/system ] ; then
-    if deb-systemd-invoke is-active arangodb3.service 2>&1 >/dev/null ; then
-        deb-systemd-invoke stop arangodb3.service >/dev/null \
-            || ar_err "failed to stop arangodb3 service" $?
+    if deb-systemd-invoke is-active arangodb3e.service 2>&1 >/dev/null ; then
```

Comment:
something is weird. In 3.3 it was also called arangodb3.service, let's not change it

Reply:
Did it actually work in 3.3?